charts/neon-broker/templates/deployment.yaml

```diff
@@ -0,0 +1,94 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "neon-broker.fullname" . }}
+  labels:
+    {{- include "neon-broker.labels" . | nindent 4 }}
+spec:
+  strategy:
+    type: Recreate
+  replicas: 1
+  # Which pods the Deployment is managing (duplicates template.metadata.labels).
+  selector:
+    matchLabels:
+      {{- include "neon-broker.selectorLabels" . | nindent 6 }}
+  # Pod template.
+  template:
+    metadata:
+      {{- with .Values.podAnnotations }}
+      annotations:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      # Labels set on created pods.
+      labels:
+        {{- include "neon-broker.selectorLabels" . | nindent 8 }}
+      {{- with .Values.podLabels }}
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+    spec:
+      {{- with .Values.imagePullSecrets }}
+      imagePullSecrets:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      serviceAccountName: {{ include "neon-broker.serviceAccountName" . }}
+      securityContext:
+        {{- toYaml .Values.podSecurityContext | nindent 8 }}
+      initContainers:
```

Comment:
Does `neon-broker` expect specific kernel tcp settings? If it does, then I suppose we can leave these here, but if not, Would perhaps make the whole `initContainers` optional, defaulting to `[]`.

Reply:
It is a proxy leftover, yes, let's remove it.